pypyr-go-terra/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# note ENTRYPOINT is still pypyr, meaning you can run this container as if it's
+# the pypyr binary.
+# be aware that the WORKDIR is NOT the GOPATH.
+# the WORKDIR is /src, which is the pypyr working dir. this will let you mount a
+# volume with a pipelines dir in it without hiding the go path.
+# build me like this from repo root:
+# docker build -t pypyr/pypyr-go-terra -t pypyr/pypyr-go-terra:latest -t pypyr/pypyr-go-terra:0.11 -t pypyr/pypyr-go-terra:0.11.8 -f pypyr-go-terra/Dockerfile .
+
+FROM pypyr/pypyr-go:pypyr1.0.0-go1.11
+
+# get the Terraform executable as zip
+RUN wget https://releases.hashicorp.com/terraform/0.11.8/terraform_0.11.8_linux_amd64.zip
+
+# install unzip
+RUN apt-get update && apt-get install -y unzip
+
+# unzip the Terraform zip file
+RUN unzip terraform_0.11.8_linux_amd64.zip
+
+# move Terraform executable to $PATH
+RUN mv terraform /usr/local/bin
+
+# remove the downloaded Terraform .zip file
+RUN rm -f terraform_0.11.8_linux_amd64.zip
```

Comment:
line 12-24 should all be a single RUN instruction, in order to minimize layers in the docker image.  i.e
```
RUN wget https://releases.hashicorp.com/terraform/0.11.8/terraform_0.11.8_linux_amd64.zip; \
         apt-get update && apt-get install -y unzip; \
         unzip terraform_0.11.8_linux_amd64.zip; \
         mv terraform /usr/local/bin; \
         rm -f terraform_0.11.8_linux_amd64.zip
```